pydarn/plotting/grid.py

```diff
@@ -0,0 +1,261 @@
+# Copyright (C) 2020 SuperDARN Canada, University of Saskatchewan
+# Author: Daniel Billett, Marina Schmidt
+#
+# Modifications:
+#
+# Disclaimer:
+# pyDARN is under the LGPL v3 license found in the root directory LICENSE.md
+# Everyone is permitted to copy and distribute verbatim copies of this license
+# document, but changing it is not allowed.
+#
+# This version of the GNU Lesser General Public License incorporates the terms
+# and conditions of version 3 of the GNU General Public License,
+# supplemented by the additional permissions listed below.
+
+
+"""
+Grid plots, mapped to AACGM coordinates in a polar format
+"""
+
+import datetime as dt
+import matplotlib.pyplot as plt
+import numpy as np
+import cmocean
```

Comment:
I ran into an issue where I needed to manually install 'cmocean' colormaps with pip before I could run code. Perhaps we should be sticking with a colormap that's already being used elsewhere? Any linear one would be okay I think, e.g. one used for power.

Reply:
yea we, just the velocity for range time was misleading